db/src/lib.rs

```diff
@@ -35,6 +42,8 @@ mod values;
 
 use edn::symbols;
 
+pub const SQLITE_MAX_VARIABLE_NUMBER: usize = 999;
```

Comment:
Note that `SQLITE_MAX_VARIABLE_NUMBER` is 500,000 on Mac OS!

We can get this value at runtime by calling `sqlite3_limit(db, 9, -1)`, where `9` is `SQLITE_LIMIT_VARIABLE_NUMBER`. I don't see that LIMIT constant exported by rusqlite.

This is the correct thing to do. Using `S_M_V_N` is expedient but won't detect a lower runtime limit imposed by `sqlite3_limit`.

Filed https://github.com/jgallagher/rusqlite/issues/220 to expose `S_M_V_N`.

Alternatively, we could consider looping over values and firing off a prepared statement over and over. I don't know which is slower: constructing the concatenated string and running one query with 999 variables, or running a small prepared statement 250 times within a transaction!